airflow/www/templates/airflow/dag.html

```diff
@@ -134,9 +134,12 @@ <h4 class="modal-title" id="taskInstanceModalLabel">
           <a id="btn_task" class="btn btn-sm" data-base-url="{{ url_for('Airflow.task') }}">
             Task Instance Details
           </a>
-          <a id="btn_rendered" class="btn btn-sm" data-base-url="{{ url_for('Airflow.rendered') }}">
+          <a id="btn_rendered" class="btn btn-sm" data-base-url="{{ url_for('Airflow.rendered_templates') }}">
             Rendered
           </a>
+          <a id="btn_rendered_k8s" class="btn btn-sm" data-base-url="{{ url_for('Airflow.rendered_k8s') }}">
+            Rendered K8s Pod Spec
```

Comment:
Can we simplify the navigation button text a bit? They're a bit verbose—maybe a word shorter. "K8s Pod Spec", "Rendered K8s Spec", "Rendered Pod Spec"?